doc/build/distributing.md

```diff
@@ -157,16 +157,6 @@ set `USE_SYSTEM_BLAS=1` and `USE_SYSTEM_LAPACK=1`, you should also set
 `LIBLAPACK=-l$(YOURBLAS)` and `LIBLAPACKNAME=lib$(YOURBLAS)`. Else, the
 reference LAPACK will be used and performance will typically be much lower.
 
-Compilation scripts
-===================
-
-The [julia-nightly-packaging](https://github.com/staticfloat/julia-nightly-packaging)
```

Comment:
Just to clarify: the above repo was last updated 7 years ago, and @staticfloat confirmed that it is not actually used anymore.